omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java

```diff
@@ -210,8 +211,16 @@ public static boolean ipMatches(String ip, List<String> candidateIps) {
 	 * @return the list of IPs
 	 */
 	public static List<String> getAllowedIps() {
-		String allowedIpsProperty = Context.getAdministrationService()
-		        .getGlobalProperty(RestConstants.ALLOWED_IPS_GLOBAL_PROPERTY_NAME, "");
+		String allowedIpsProperty = "";
+		try {
+			Context.addProxyPrivilege("Get Global Properties");
```

Comment:
Why not use `PrivilegeConstants`?

Reply:
`PrivilegeConstants.GET_GLOBAL_PROPERTIES` is not available in the platform version that this module uses

Reply:
Which version is it?